app/try_ruby.rb

```diff
@@ -61,7 +62,8 @@ def initialize
 
     #If hold down the control and the Enter key goes down, run
     $document.on :keydown, '#editor' do |e|
-      if e.key == "Enter" && e.ctrl?
+      if (e.key == "Enter" && (e.ctrl? && !Helper.ios?)) || ((Helper.macos? || Helper.ios?) && e.meta?)
```

Comment:
Right-hand side of the || expression is missing check for `Enter`.

Reply:
Thank you for pointing this out. I've fixed the issue, please check it out.